src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -427,9 +428,9 @@ export class QuickBar extends LitElement {
         `ui.dialogs.quick-bar.commands.navigation.${page.component}`
       );
 
-      if (page.translationKey) {
+      if (page.translationKey && shortCaption) {
         const caption = this.hass.localize(
-          "ui.dialogs.quick-bar.commands.navigation.navigate_to_config",
```

Comment:
Should <https://github.com/home-assistant/frontend/pull/7753/files#diff-e67939fd25c650222db710f18764d10ae69454b0e8ad680f5e10177c2db93ceaR509> be removed now?

Reply:
Weird, I thought I had already.